### ISSUE TITLE
TypeError: navigator.plugins is not iterable

### ISSUE THREAD

**Issue by AJGranowski** [sereneblue/chameleon#555]
## Expected Behavior

```js
for (const plugin of navigator.plugins) {console.log(plugin.name);}
```

should print a list of plugin names, for example:

```
PDF Viewer
Chrome PDF Viewer
Chromium PDF Viewer
Microsoft Edge PDF Viewer
WebKit built-in PDF
```

## Current Behavior

```js
for (const plugin of navigator.plugins) {console.log(plugin.name);}
```

results in the following error:

```
Uncaught TypeError: navigator.plugins is not iterable
    <anonymous> debugger eval code:1
```

However,

```js
console.log(navigator.plugins[0].name);
```
will print `"PDF Viewer"`...

## Relevant settings

Unknown. Probably related to https://github.com/sereneblue/chameleon/blob/d7bc873cd706888a414fe9c0c3de34d56a4f88e8/src/lib/inject.ts#L341-L446

## Context (Environment)

Firefox 109.0 (64-bit) on Windows 10

**Comment by AJGranowski** [comment]
I could not find a setting besides enable/disable Chameleon that affects this. Disabling Chameleon results in the error disappearing.

**Comment by sereneblue** [comment]
Hi @AJGranowski, thanks for reporting. This issue has been resolved in v0.22.40.
